mne/channels/montage.py

```diff
@@ -606,7 +606,7 @@ def _get_montage_in_head(montage):
 
 
 @fill_doc
-def _set_montage(info, montage, match_case=True):
+def _set_montage(info, montage, match_case=True, on_missing=True):
```

Comment:
@adam2392 why do you keep resolving this conversation? we both agree with @larsoner that on_missing takes string values and has 3 options. It should not be a book anywhere

Reply:
ah sorry, I added the option:

```
        _check_option('on_missing', on_missing, ['raise',
                                                 'ignore',
                                                 'warn'])
```

and forgot to change the function header.